tools/coredump/testdata/amd64/alpine320-nobuildid.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
perhaps rename the file to `python3-alpine320-nobuildid.json` or similar to include the interpreter in the name